core/server/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -262,10 +441,19 @@ public InodeDirectory getRoot() {
     return mRoot;
   }
 
+  // package private for testing.
+  CreatePathResult createPath(AlluxioURI path, CreatePathOptions<?> options)
```

Comment:
move to the test subtree and use WhiteBox.invokeMethod() to call lockInodePath() and createPath()